fapolicy_analyzer/ui/main_window.py

```diff
@@ -15,6 +15,8 @@
 
 import logging
 import os
+import json
+import subprocess
```

Comment:
I dont think subprocess should be used for anything in this PR

Reply:
> I dont think subprocess should be used for anything in this PR

I was too broad with this, it shouldn't be used for anything at runtime.  Build time is good, which is what is being done currently.  :+1: